pkg/pci/pci_linux.go

```diff
@@ -44,6 +44,28 @@ func getDeviceModaliasPath(ctx *context.Context, address string) string {
 	)
 }
 
+func getDeviceRevision(ctx *context.Context, address string) string {
+	paths := linuxpath.New(ctx)
+	pciAddr := AddressFromString(address)
+	if pciAddr == nil {
+		return ""
+	}
+	revisionPath := filepath.Join(
+		paths.SysBusPciDevices,
+		pciAddr.Domain+":"+pciAddr.Bus+":"+pciAddr.Slot+"."+pciAddr.Function,
+		"revision",
+	)
+
+	if _, err := os.Stat(revisionPath); err != nil {
+		return ""
+	}
```

Comment:
this is not wrong, but it seems we can just call ReadFile directly instead of checking with `os.Stat` before.

Reply:
I concur. I implemented it in this fashion to mirror the existing approach for reading the modalias file, for the sake of consistency. I'm not against the simplified approach of attempting a read and reacting on the returned error

Reply:
I don't have strong opinions here. Both approaches have their merits.

Reply:
yeah, totally fine to use existing pattern here.